{{cookiecutter.repo_name}}/{{cookiecutter.module_name}}/tests/test_plugin.py

```diff
@@ -0,0 +1,102 @@
+# -----------------------------------------------------------------------------
+# Copyright (c) 2014--, The Qiita Development Team.
+#
+# Distributed under the terms of the BSD 3-clause License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# -----------------------------------------------------------------------------
+
+from unittest import TestCase, main
+from tempfile import mkdtemp
+from os import remove, environ
+from os.path import exists, isdir
+from shutil import rmtree
+from json import dumps
+
+from qiita_client import QiitaClient
+
+from {{cookiecutter.module_name}}.plugin import execute_job
+
+
+CLIENT_ID = '19ndkO3oMKsoChjVVWluF7QkxHRfYhTKSFbAVt8IhK7gZgDaO4'
```

Comment:
This is only used for the tests - since this CLIENT_ID/CLIENT_SECRET pair is already in the test database makes testing really easy, since it never changes. There is nothing in the code to prevent 2 plugins having the same CLIENT_ID/CLIENT_SECRET and the only reason why we are generating a pair for each plugin is in order to be able to keep track of statistics in the future.